app/components/footer/cookie_acceptance_component.html.erb

```diff
@@ -9,19 +9,29 @@
           We also share some of this information with our social media, advertising and analytics partners.
         </p>
       </div>
-
       <div class="dialog__buttons">
         <!--
           The id is `biscuits-agrees` instead of `cookies-agree`
           to prevent ad blockers from removing the button.
         -->
-        <a tabindex="-1" href="#" class="button call-to-action-button" data-action="click->cookie-acceptance#accept" data-cookie-acceptance-target="agree" id="biscuits-agree">
-          <span>Accept all cookies</span>
-        </a>
-
-        <%= link_to(cookie_preference_path, tabindex: -1, id: "cookies-disagree", class: "button button--secondary", data: { "cookie-acceptance-target": "disagree" }) do %>
-          <span>Set cookie preferences</span>
-        <% end %>
+        <%= render(Content::LinkAsButtonComponent.new(
+            link_text: "Accept all cookies",
+            url: "#",
+            options: {
+              tabindex: -1,
+              id: "biscuits-agree",
+              class: "call-to-action-button",
+              data: { action: "click->cookie-acceptance#accept", "cookie-acceptance-target": "agree" },
+            } )) %>
+        <%= render(Content::LinkAsButtonComponent.new(
+            link_text: "Set cookie preferences",
+            url: cookie_preference_path,
+            options: {
```

Comment:
I'm not sure of the value of wrapping the optional parameters in the `options {}` hash - maybe cleaner to just make them conventional optional parameters ?